packages/dd-trace/src/appsec/iast/analyzers/vulnerability-analyzer.js

```diff
@@ -35,7 +37,11 @@ class Analyzer extends Plugin {
 
   analyze (value) {
     const iastContext = getIastContext(storage.getStore())
-    if (iastContext && this._isVulnerable(value, iastContext) && this._checkOCE(iastContext)) {
+    if (!iastContext) {
+      return
+    }
+
+    if (this._isVulnerable(value, iastContext)) {
```

Comment:
```suggestion
    if (iastContext && this._isVulnerable(value, iastContext)) {
```
And remove the previous check